webapp/channels/src/components/user_list_row_with_error/user_list_row_with_error.tsx

```diff
@@ -89,12 +88,13 @@ export default class UserListRowWithError extends React.PureComponent<Props, Sta
         if (this.props.user.is_bot) {
             email = null;
         } else if (this.props.extraInfo && this.props.extraInfo.length > 0) {
+            // Replaced FormattedMarkdownMessage with FormattedMessage
             email = (
-                <FormattedMarkdownMessage
-                    id='admin.user_item.emailTitle'
+                <FormattedMessage
+                    id='admin.user_item.emailTitle_updated'
                     defaultMessage='**Email:** {email}'
```

Comment:
i wouldnt want to give this solution yet, but since i understand this being a bit tricky 
```
                <FormattedMessage
                    id='admin.user_item.emailTitle_updated'
                    defaultMessage='<b>Email:</b> {email}'
                    values={{
                        b: (.....)
                        email: this.props.user.email,
                    }}
                    ```